vendor/github.com/onsi/gomega/go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/onsi/gomega
 
+go 1.14
```

Comment:
Hmm... really? No idea how that would have gotten in there. The tests all pass so figuring this is ok...

Reply:
The fact that only this `go.mod` file, out of all the ones in `vendor/`, has been updated makes me think that your editor has done something weird. Revert this probably?

Reply:
Yeah lets revert it 👍